saleor/shipping/models.py

```diff
@@ -84,7 +84,7 @@ def __str__(self):
     def countries_display(self):
         countries = self.countries
         if self.default:
-            from ..dashboard.shipping.forms import get_available_countries
+            from .utils import get_available_countries
```

Comment:
Are you sure that circular imports still exist? Maybe we could move this import at the beginning of the file?

Reply:
I checked, circular imports still exists after moving this import to beginning of the file. `saleor/shipping/utils` imports `ShippingZone` from `saleor/shipping/models.py`. We can drop this import by moving `currently_used_countries` and `get_available_countries` functions from `utils.py` to `models.py` in shipping. But I am not sure if it's a good solution.

Reply:
The functions referencing the `ShippingZone` and being used by `ShippingZone` are query-set methods/utilities. They should definitively be inside the `QuerySet` manager of `ShippingZone`

Reply:
But `currently_used_countries` and `get_available_countries` operate on all existing `ShippingZone` objects, not on specific (e.g. filtered) queryset. So I'm not sure if putting this functions in `QuerySet` manager is a good solution.

Reply:
Yes, but it is returning a query-set. Having all query-set filtering is better in a single same place (manager). Just like we do for `visible_to_user` (there is `get_visible_to_user` too looks like 😕).

Personally, when working on a django project, I prefer finding all the filtering operations in a single same place. But we don't have any convention for that currently... We are doing both query-set and `utils.py`.

Maybe @maarcingebala or @korycins have an input about it?

Reply:
Do we even need this method anymore? It was used only in old dashboard
<img width="379" alt="Screenshot 2019-11-25 at 15 34 12" src="https://user-images.githubusercontent.com/5188636/69549168-0d8d5300-0f99-11ea-8435-b285f6ada07c.png">

Reply:
Do we need this method anymore? It seems to be used only in dashboard we want to drop.
<img width="379" alt="Screenshot 2019-11-25 at 15 34 12" src="https://user-images.githubusercontent.com/5188636/69549423-91dfd600-0f99-11ea-9399-0ed215a9eadf.png">

Reply:
Oh, good point! Then `countries_display` could be removed, also `get_available_countries` and `currently_used_countries`will be not needed anymore. So I will remove those and there will be no problem anymore in this case. 😌